event_webhook_types.go

```diff
@@ -618,6 +618,7 @@ type MergeEvent struct {
 		MilestoneID              int          `json:"milestone_id"`
 		StateID                  StateID      `json:"state_id"`
 		State                    string       `json:"state"`
+		Draft                    bool         `json:"draft"`
```

Comment:
Can you please place the field below `WorkInProgress` following the ordering of the docs?

Reply:
True!